src/components/project/project.resolver.ts

```diff
@@ -312,6 +312,40 @@ export class ProjectResolver {
     );
   }
 
+  @ResolveField(() => SecuredPeriodicReport, { nullable: true })
+  async currentFinancialReport(
+    @AnonSession() session: Session,
+    @Parent() project: Project
+  ): Promise<SecuredPeriodicReport> {
+    const value = await this.periodicReportService.getCurrentReport(
+      project.id,
+      ReportType.Financial,
+      session
+    );
+    return {
+      canRead: true,
+      canEdit: true,
```

Comment:
```suggestion
      canEdit: false,
```
This prop cannot be modified since it's calculated.